spec/features/create_receipt_spec.rb

```diff
@@ -0,0 +1,36 @@
+feature 'As a customer I want to be able to order items from a coffee shop' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.